src/Configuration/SqliteAdapter.php

```diff
@@ -1,20 +1,21 @@
-<?php namespace Nord\Lumen\Doctrine\ORM\Configuration;
+<?php
+
+namespace Nord\Lumen\Doctrine\ORM\Configuration;
 
 use Nord\Lumen\Doctrine\ORM\Contracts\ConfigurationAdapter as ConfigurationAdapterContract;
 
 class SqliteAdapter implements ConfigurationAdapterContract
 {
-
     /**
-     * @inheritdoc
+     * {@inheritdoc}
```

Comment:
I expect it is configurable in StyleCI, although we're [using the PSR-2 coding style guide](https://github.com/nordsoftware/lumen-doctrine/blob/develop/.github/CONTRIBUTING.md) so [StyleCI is using that preset](https://github.com/nordsoftware/lumen-doctrine/blob/develop/.styleci.yml) (and this PR was automatically sent by StyleCI). 

Perhaps it'd make more sense to configure PhpStorm to also follow PSR-2 ([e.g.](http://www.matthewsetter.com/enforcing-psr1-and-psr2-with-phpstorm/)).